pkg/handler/websocket/common.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package websocket
+
+import (
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// writes close control message to the websocket connection. Same as the default websocket close handler.
+func writeCloseMessage(ws *websocket.Conn, code int) {
+	message := websocket.FormatCloseMessage(code, "")
+	_ = ws.WriteControl(websocket.CloseMessage, message, time.Now().Add(time.Second))
```

Comment:
I guess this returns potential err, wouldn't it be better to return it instead of returning `nil`?

Reply:
you are right, i did it automatically like websocket does it, returning err is better